index.js

```diff
@@ -1,4 +1,4 @@
-const writeConf = require('./write-config');
+const readConf = require('gh-conf/gh-conf-write');
```

Comment:
You need to change this to 
`const writeConf = require('./write-config');`

Then all the test cases will pass